web/pwa/components/ActivityRevision/ActivityRevision.js

```diff
@@ -323,6 +324,9 @@ export default function ActivityRevisionOrCreationModal({
               .filter(a =>
                 allowSupportActivity ? true : a !== ACTIVITIES.support.name
               )
+              .filter(a =>
+                allowTransfers ? true : a !== ACTIVITIES.transfer.name
+              )
```

Comment:
Can't we use only one filter here?

```suggestion
              .filter(a =>
                allowSupportActivity ? true : a !== ACTIVITIES.support.name &&
                allowTransfers ? true : a !== ACTIVITIES.transfer.name
              )
```

Reply:
Yes we can !